Logger/LogFactory.cs

```diff
@@ -1,11 +1,31 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public class LogFactory
     {
+        private string FilePath;
```

Comment:
Fields should be named underscore PascalCase
```suggestion
        private string _FilePath;
```